package/harmony/package

```diff
@@ -25,3 +25,7 @@ package() {
     mkdir -p "$pkgdir"/opt/etc/draft/
     install -D -m 644 "$srcdir"/src/harmony/harmony.draft "$pkgdir"/opt/etc/draft/harmony.draft
 }
+
+configure() {
+    mkdir -p /home/root/harmony 2> /dev/null
```

Comment:
This redirect to `/dev/null` is not needed since `mkdir -p` does not print anything, even if the directory already exists.

```suggestion
    mkdir -p /home/root/harmony
```